security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,54 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
 
 For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
 
+
+[id="compliance-operator-release-notes-1-7-0_{context}"]
+== OpenShift Compliance Operator 1.7.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.7.0:
+
+* link:https://access.redhat.com/errata/RHBA-2025:3728[RHBA-2025:3728 - OpenShift Compliance Operator 1.7.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-7-0-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* A `must-gather` extension is now available for the Compliance Operator installed on `aarch64`, `x86`, `ppc64le`, and `s390x` architectures. The `must-gather` tool provides crucial configuration details to Red Hat Customer Support and engineering. For more information, see xref:../../security/compliance_operator/co-support.adoc#compliance-must-gather_co-support[Using the must-gather tool for the Compliance Operator].
```

Comment:
The parentheses around all of the issues need removed. It renders as in parentheses in the preview. So it's like "For more information, see (bug)"

Reply:
@GroceryBoyJr The parentheses is common, but it's the lead in here that stands out:

![Screenshot From 2025-04-28 14-58-47](https://github.com/user-attachments/assets/88f690b5-9854-43b3-8738-0cf8fb84949d)

vs 

![Screenshot From 2025-04-28 14-59-11](https://github.com/user-attachments/assets/51f146e8-442d-4b88-9aa1-c3824269ffb1)